src/libcore/atomic.rs

```diff
@@ -285,7 +286,8 @@ impl AtomicBool {
     ///
     /// let some_bool = AtomicBool::new(true);
     ///
-    /// let value = some_bool.store(false, Ordering::Relaxed);
+    /// assert_eq!(true, some_bool.compare_and_swap(false, false, Ordering::Relaxed));
```

Comment:
I prefer `assert_eq!(actual, expected)` as well, but all other examples for `AtomicBool` used `assert_eq!(expected, actual)` and I wanted to stay consistent. Should I swap the arguments of all other asserts in this file as well?